cds_books/config.py

```diff
@@ -243,3 +245,69 @@ def _parse_env_bool(var_name, default=None):
                        "invenio_opendefinition"]
 
 ILS_LITERATURE_COVER_URLS_BUILDER = build_syndetic_cover_urls
+
+# Namespaces for fields added to the metadata schema
+ILS_RECORDS_METADATA_NAMESPACES = {
+    "document": {
+        "accelerator_experiments": {
+            "@context": "https://test.org/accelerator_experiments/terms"
```

Comment:
I would replace this with `https://greybook.cern.ch/greybook/` or simply `https://cern.ch/` to make clear that this is a CERN thingy, same below